File: exs/mundo_1/ruby/025.rb
=begin
Desafio 025

Problema: Crie um programa que leia o nome de uma pessoa e diga se ela tem "SILVA" no nome.

Resolução do problema:
=end

print"Digite seu nome: "
n = gets.chomp.upcase

if n.index('SILVA') != nil
	puts"Você tem Silva no nome." 
else
	puts"Você não tem Silva no nome."
end

File: exs/mundo_1/golang/025.go
/*
Desafio 025

Problema: Crie um programa que leia o nome de uma pessoa e diga se ela tem "SILVA" no nome.

Resolução do problema:
*/

package main

import (
	"bufio"
	"fmt"
	"strings"
	"os"
)

func main(){
	scanner := bufio.NewReader(os.Stdin)
	fmt.Print("Digite seu nome: ")
	nome, _ := scanner.ReadString('\n')

	if strings.Contains(strings.ToUpper(nome), "SILVA"){
		fmt.Println("Seu nome tem Silva.")
	} else {
		fmt.Println("Seu nome não tem Silva.")
	}
}
